docs/pages/get-started/installation.mdx

```diff
@@ -24,7 +24,7 @@ To use Expo, you need to have the following tools installed on your machine:
 
 After installing Node.js, you can use `npx` to create a new app.
 
-<Terminal cmd={['$ npx create-expo-app --template']} />
+<Terminal cmd={['$ npx create-expo-app my-app']} />
```

Comment:
Hey @rudolfolah,  I think your suggestion about adding the context as to what to expect when running the `npx create-expo-app --template` sounds good to me (instead of recommending the `my-app` version which just creates a blank template). How about adding the context after this line?